README.md

```diff
@@ -1,44 +1,39 @@
 
-> **Note**: These Recipes are automatically installed via `rad init`
+  - [Applications.Datastores/redisCaches](https://docs.radapp.io/reference/resource-schema/cache/redis/)
+  - [Applications.Datastores/mongoDatabases](https://docs.radapp.io/reference/resource-schema/databases/mongodb/)
+  - [Applications.Datastores/sqlDatabases](https://docs.radapp.io/reference/resource-schema/databases/microsoft-sql/)
+  - [Applications.Messaging/rabbitmqQueues](https://docs.radapp.io/reference/resource-schema/messaging/rabbitmq/)
+  - [Applications.Dapr/stateStores](https://docs.radapp.io/reference/resource-schema/dapr-schema/statestore/)
+  - [Applications.Dapr/pubSubBrokers](https://docs.radapp.io/reference/resource-schema/dapr-schema/pubsub/)
+  - [Applications.Dapr/secretStores](https://docs.radapp.io/reference/resource-schema/dapr-schema/secretstore/)
+  - [Applications.Core/extenders](https://docs.radapp.io/reference/resource-schema/core-schema/extender/)
 
-| Recipe | Resource | Description | Template Path |
-|--------|----------|-------------|---------------|
-| [`local-dev/daprpubsubbrokers`](/local-dev/pubsubbrokers.bicep) | `Applications.Dapr/pubSubBrokers` | A lightweight container running the `redis` image and a Redis Dapr Pub/Sub component for development purposes. | `ghcr.io/radius-project/recipes/local-dev/daprpubsubbrokers:TAG` |
-| [`local-dev/daprstatestores`](/local-dev/statestores.bicep) | `Applications.Dapr/stateStores` |A lightweight container running the `redis` image and a Redis Dapr state store component for development purposes. | `ghcr.io/radius-project/recipes/local-dev/daprstatestores:TAG` |
-| [`local-dev/extender-postgresql`](/local-dev/extender-postgresql.bicep) | `Applications.Core/extenders` |A lightweight container running the `postgres` image for development purposes. Used with the Radius extender resource. | `ghcr.io/radius-project/recipes/local-dev/extender-postgresql:TAG` |
-| [`local-dev/rabbitmqmessagequeues`](/local-dev/rabbitmqmessagequeues.bicep) | `Applications.Messaging/rabbitMQQueues` |A lightweight container running the `rabbitmq` image for development purposes. | `ghcr.io/radius-project/recipes/local-dev/rabbitmqmessagequeues:TAG` |
-| [`local-dev/rediscaches`](/local-dev/rediscaches.bicep) | `Applications.Datastores/redisCaches` |A lightweight container running the `redis` image for development purposes. | `ghcr.io/radius-project/recipes/local-dev/rediscaches:TAG` |
-| [`local-dev/mongodatabases`](/local-dev/mongodatabases.bicep) | `Applications.Datastores/mongoDatabases` |A lightweight container running the `mongo` image for development purposes. | `ghcr.io/radius-project/recipes/local-dev/mongodatabases:TAG` |
-| [`local-dev/sqldatabases`](/local-dev/sqldatabases.bicep) | `Applications.Datastores/sqlDatabases` |A lightweight container running the `azure-sql-edge` image for development purposes. | `ghcr.io/radius-project/recipes/local-dev/sqldatabases:TAG` |
+## Supported Cloud Providers
```

Comment:
With Terraform recipes, we don't really block other cloud providers, we just don't have first class support for configuring the provider, so I'm wondering if we should explicitly mention that.

Also, this makes me think that we don't mention Terraform in this repo anywhere, do we need to call out the rationale behind keeping it limited to bicep recipes at this point?

@AaronCrawfis, thoughts?

Reply:
> do we need to call out the rationale behind keeping it limited to bicep recipes at this point?

I see adding Terraform recipes is covered here https://github.com/radius-project/recipes/blob/8cbb91a4f00d835a1bf52e96b121528dee347162/docs/contributing/contributing-recipes.md?plain=1#L9, so you can ignore my second question above :)

Reply:
Still curious to hear thoughts on how to approach documentation on the previous I had though - 

"With Terraform recipes, we don't really block other cloud providers, we just don't have first class support for configuring the provider, so I'm wondering if we should explicitly mention that."

Reply:
Just chatted with Reshma, we're going to pull out all the Recipe specific sections and point users toward https://docs.radapp.io/guides/recipes/overview/